packages/client/components/ReflectionGroup/ReflectionGroup.tsx

```diff
@@ -92,22 +93,19 @@ const ReflectionGroup = (props: Props) => {
   const {phaseType} = localPhase
   const {isComplete} = localStage
   const {reflections, id: reflectionGroupId, titleIsUserDefined} = reflectionGroup
-  const isSpotlightSrcGroup = spotlightGroup?.id === reflectionGroupId
-  const isSpotlightOpen = !!spotlightGroup?.id
-  const visibleReflections = useMemo(() => {
-    return isSpotlightSrcGroup
-      ? reflections.filter(({id}) => !reflectionIdsToHide?.includes(id))
-      : reflections.slice()
-  }, [reflections, reflectionIdsToHide])
-  const isInSpotlight = !openSpotlight
-  const isBehindSpotlight = isSpotlightOpen && !isInSpotlight
-  const isDroppable = useMemo(() => {
-    const isSourceGroup = spotlightGroup?.id === reflectionGroupId
-    const isDraggingSource = !!spotlightGroup?.reflections.find(
-      ({isViewerDragging}) => isViewerDragging
-    )
-    return isSpotlightOpen ? isDraggingSource || isSourceGroup : true // prevent grouping results into results
-  }, [spotlightGroup])
+  const spotlightGroupId = spotlightGroup?.id
+  const visibleReflections = useMemo(
+    () => reflections.filter(({id}) => !reflectionIdsToHide?.includes(id)),
+    [reflections, reflectionIdsToHide]
+  )
+  const isSpotlightSrcGroup = spotlightGroupId === reflectionGroupId
+  const isBehindSpotlight = !!(spotlightGroupId && openSpotlight)
+  const [hideSpotlightHeader, disableDrop] = useSpotlightReflectionGroup(
```

Comment:
I felt that using a hook to tidy away some of the logic would be better than creating a separate `SpotlightReflectionGroup` file because:
1. The kanban `reflectionGroup` would still need to know if the Spotlight is open for the `disableDrop` logic as well as the `isHiddenSpotlightSource` logic
2. There would be a lot of repeated code
3. With the hook, there's only 12 Spotlight related lines, some of which would be needed anyway due to point 1

I don't love that `hideSpotlightHeader` & `disableDrop` are largely unrelated but living in the same hook. I considered separating them but they both require `isRemoteSpotlightSrc` and keeping them in one hook reduces the length of the logic here.